recorder/src/main/cpp/prob_pct.cc

```diff
@@ -0,0 +1,17 @@
+#include "prob_pct.hh"
+#include <fstream>
+
+ProbPct::ProbPct() {
+    std::ifstream in("/dev/urandom", std::ios_base::in | std::ios_base::binary);
+    std::uint8_t b;
+    for (auto i = 0; i < SZ;) {
+        in >> b;
+        if (b >= 200) continue;
```

Comment:
Bigger the domain, better the randomization (with perfect scaling it is equivallent to 100), but for that we need to take the raw feed and scale it ourselves, which is what we do..